addons/l10n_cn/account_chart_small_business_template.xml

```diff
@@ -252,7 +252,7 @@ http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/t20111107_605525.html
             <field name="user_type" ref="user_type_bs39"/>
         </record>
 
-        <record id="small_business_chart1602" model="account.account.template">
+        <record id="small_business_chart1622" model="account.account.template">
```

Comment:
If we don't change this one, there will be two 'small_business_chart1602' in this file, the first will be override by this one, If don't change this, Maybe I will change the first one?